app/build.gradle.kts

```diff
@@ -181,26 +181,33 @@ dependencies {
     globalTestImplementation(libs.compose.test.junit)
     globalTestImplementation(libs.kaspresso)
     globalTestImplementation(libs.kaspresso.compose)
-    globalTestImplementation(libs.junit)
```

Comment:
Warning: I've seen you put `libs.junit` in both test and androidTest implementations. You can keep it to global then